src/app/app.module.ts

```diff
@@ -3,10 +3,12 @@ import { NgModule } from '@angular/core';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { OpportunityDivideComponent } from './opportunity-divide/opportunity-divide.component';
 
 @NgModule({
   declarations: [
-    AppComponent
+    AppComponent,
+    OpportunityDivideComponent
```

Comment:
You should have pulled the YearUpSolution component from master so you won't have a conflict in the file here.